src/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/6aed09d264.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>To-Do-List</title>
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <div class="heading">Today's To Do</div>
+      <input type="text" id="input_text" />
```

Comment:
- [ ] The input new items container should have an enter button or icon that can be clicked on to input new tasks to the items. 😉